src/main/java/net/sf/javaanpr/imageanalysis/PlateHorizontalGraph.java

```diff
@@ -25,46 +25,42 @@
     private static int horizontalDetectionType =
             Configurator.getConfigurator().getIntProperty("platehorizontalgraph_detectionType");
 
-    private Plate handle;
-
-    public PlateHorizontalGraph(Plate handle) {
-        this.handle = handle;
-    }
-
     public float derivation(int index1, int index2) {
         return this.yValues.elementAt(index1) - this.yValues.elementAt(index2);
     }
 
-    public Vector<Peak> findPeak(int count) {
+    public Vector<Peak> findPeak() {
         if (PlateHorizontalGraph.horizontalDetectionType == 1) {
-            return this.findPeak_edgedetection(count);
+            return this.findPeak_edgedetection();
         }
-        return this.findPeak_derivative(count);
+        return this.findPeak_derivative();
     }
 
-    public Vector<Peak> findPeak_derivative(int count) {
```

Comment:
Optional: Would you formatting the name of this and the other `findPeak` method to camelCase?